Makefile

```diff
@@ -42,6 +48,9 @@ PKG_CONFIG ?= pkg-config
 CFLAGS ?= -ansi -O0 -g3 -Wall -Wextra -Werror -Wconversion \
 	-Wstrict-prototypes -Wno-unused-parameter -pedantic
 CFLAGS += -fPIC -DMPACK_DEBUG_REGISTRY_LEAK
+ifeq ($(PLATFORM), Darwin)
+CFLAGS += -std=c99
```

Comment:
any reason to make this conditional on Darwin?

Reply:
@justinmk no

The short answer is I wanted to minimize the impact of my changes, but I have no particular reason to think there'd be any issue with adding this for both linux and osx.

I'm happy to update the PR to add this flag unconditionally -- want me to?

Also thanks for taking a look!

Reply:
> I'm happy to update the PR to add this flag unconditionally -- want me to?

I'd say yes

Reply:
@justinmk sounds good, I've updated the PR